pyproject.toml

```diff
@@ -48,7 +48,7 @@ cov = [
     "cov-report",
 ]
 e2e = [
-    "git submodule add --force https://github.com/open-feature/spec.git spec",
+    "git submodule update --init --recursive",
```

Comment:
This isn't necessary when running in a GitHub action but it's probably worth keeping for local development.